sar.drush.inc

```diff
@@ -148,6 +186,27 @@ function drush_sar_replace() {
   }
 }
 
+/**
+ * Command callback function for drush sarj.
+ *
+ * Search and replace all strings in JS Injector rules.
+ */
+function drush_sar_replace_js_injector() {
+  // Parse our command line options.
+  $options = drush_sar_parse_options();
+
+  // If there's a problem yo I'll solve it...
```

Comment:
check out the hook while my Dev revolves it.

Reply:
`s/revolves/resolves/`